abi.js

```diff
@@ -2,6 +2,19 @@ var gypinstall = require('./gypinstall')
 var util = require('./util')
 var error = require('./error')
 
+function getAbiFromTarget (target) {
+  if (!target) return process.versions.modules
+  if (target === process.versions.node) return process.versions.modules
+  if (/^7\.[0-9]+\.[0-9]+/.test(target)) return '51'
+  if (/^6\.[0-9]+\.[0-9]+/.test(target)) return '48'
+  if (/^5\.[0-9]+\.[0-9]+/.test(target)) return '47'
+  if (/^4\.[0-9]+\.[0-9]+/.test(target)) return '46'
+  if (/^0\.12\.[0-9]+/.test(target)) return '14'
+  if (/^0\.10\.[0-9]+/.test(target)) return '11'
```

Comment:
what about simplifying more to this?

- `/^7\./`
- `/^6\./`
- `/^5\./`
- `/^4\./`
- `/^0\.12\./`
- `/^0\.10\./`